vsc_pdf_template_editor/lib/src/widgets/tree_view_widget.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
+import 'package:flutter_treeview/flutter_treeview.dart';
+import '../stores/tree_store.dart';
+
+class TreeViewWidget extends StatelessWidget {
+  const TreeViewWidget({
+    Key? key,
+    required this.viewModel,
+  }) : super(key: key);
+
+  final TreeStore viewModel;
+
+  @override
+  Widget build(BuildContext context) {
+    final width = MediaQuery.of(context).size.width;
+    final height = MediaQuery.of(context).size.height;
+    return LayoutBuilder(
+      builder: (context, constraint) {
+        return SingleChildScrollView(
+          child: ConstrainedBox(
+              constraints: BoxConstraints(minHeight: constraint.maxHeight),
+              child: SizedBox(
+                width: width * 0.25,
+                height: height - 50, //TODO add smart resolving
```

Comment:
In a follow-up PR, let's figure-out why this forced-sizing is here. Seems like it should just take all of the height it wants because it's in a scroll view.  Also, the width should be constrained by it's parent.